cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -134,14 +140,45 @@ def migration(self, key, value):
                 "subscription": clean_val("a", value, str),
             }
         )
-    if key == "780__":
-        _relation_previous = clean_val("w", value, str, req=True)
 
-    if key == "785__":
-        _relation_next = clean_val("w", value, str, req=True)
+    try:
+        _related = _migration.get("related", [])
+        relation_type = None
+        _has_related = _migration.get("has_related", False)
+        # language relation
+        if key == "787__" and "i" in value:
+            relation_language = clean_val("i", value, str)
+            if relation_language:
+                relation_type = LANGUAGE_RELATION.name
+        # has supplement/supplement to relation
+        if key == "770__" or key == "772__" and "i" in value:
+            clean_val("i", value, str)
```

Comment:
the `i` subfield should be translated to the `note` field of the relation type `other`